controls/anssi.yml

```diff
@@ -17,489 +17,362 @@ levels:
-      The second involves removal of any default secret or trusted element
-      configured by the operating system during install process, e.g. default
-      known passwords.
-    automated: no
-
-  - id: R21
-    levels:
-    - intermediary
-    title: Hardening and monitoring of services subject to arbitrary flows
-    notes: >-
-      SELinux can provide confinement and monitoring of services, and AIDE provides
-      basic integrity checking. System logs are configured as part of R43.
-      Hardening of particular services should be done on a case by case basis and is
-      not automated by this content.
-    status: partial
-    rules:
-    - selinux_state
-    - var_selinux_state=enforcing
-    - package_aide_installed
-    - aide_build_database
-
-  - id: R22
-    levels:
-    - intermediary
-    title: Setting up network sysctl
-    status: automated
-    rules:
-    # No routing between interfaces
-    # net.ipv4.ip_forward = 0
-    - sysctl_net_ipv4_ip_forward
-
-    # Reverse path filtering
-    # net.ipv4.conf.all.rp_filter = 1
-    - sysctl_net_ipv4_conf_all_rp_filter
-
-    # net.ipv4.conf.default.rp_filter = 1
-    - sysctl_net_ipv4_conf_default_rp_filter
-
-    # Do not send ICMP redirects
-    # net.ipv4.conf.all.send_redirects = 0
-    - sysctl_net_ipv4_conf_all_send_redirects
-
-    # net.ipv4.conf.default.send_redirects = 0
-    - sysctl_net_ipv4_conf_default_send_redirects
-
-    # Deny source routing packets
-    # net.ipv4.conf.all.accept_source_route = 0
-    - sysctl_net_ipv4_conf_all_accept_source_route
-
-    # net.ipv4.conf.default.accept_source_route = 0
-    - sysctl_net_ipv4_conf_default_accept_source_route
-
-    # Do not accept ICMPs of redirect type
-    # net.ipv4.conf.all.accept_redirects = 0
-    - sysctl_net_ipv4_conf_all_accept_redirects_value=disabled
-    - sysctl_net_ipv4_conf_all_accept_redirects
-
-    # net.ipv4.conf.all.secure_redirects = 0
-    - sysctl_net_ipv4_conf_all_secure_redirects
-
-    # net.ipv4.conf.default.accept_redirects = 0
-    - sysctl_net_ipv4_conf_default_accept_redirects_value=disabled
-    - sysctl_net_ipv4_conf_default_accept_redirects
-
-    # net.ipv4.conf.default.secure_redirects = 0
-    - sysctl_net_ipv4_conf_default_secure_redirects
-
-    # Log packets with abnormal IPs
-    # net.ipv4.conf.all.log_martians = 1
-    - sysctl_net_ipv4_conf_all_log_martians
-
-    # RFC 1337
-    # net.ipv4.tcp_rfc1337 = 1
-    - sysctl_net_ipv4_tcp_rfc1337
-
-    # Ignore responses that do not comply with RFC 1122
-    # net.ipv4.icmp_ignore_bogus_error_responses = 1
-    - sysctl_net_ipv4_icmp_ignore_bogus_error_responses
-
-    # Increase the range for ephemeral ports
-    # net.ipv4.ip_local_port_range = 32768 65535
-    - sysctl_net_ipv4_ip_local_port_range
-
-    # Use SYN cookies
-    # net.ipv4.tcp_syncookies = 1
-    - sysctl_net_ipv4_tcp_syncookies
+      When IPv6 is not in use, disable it. Otherwise secure the entire IPv6 as well.
+    status: automated
+    rules:
```

Comment:
In my option we should only have the rules that secure IPv6. It's long past time to always disable IPv6.

Reply:
I understand, but still, the principle of minimizing attack surface applies.
If one doesn't use IPv6, why have it enabled?

The  question is whether the profile should be opinionated about it.

Reply:
I agree that if you not use it disable it. We have to take stand one or the another for this profile and I think we should go with the secure IPv6.

Reply:
It is true that in the V1.2 profile we rather configured ipv6 than disabled it, afaik.
I am attaching a diff of selected rules, it can be seen there clearly.
[diff_rhel9.txt](https://github.com/ComplianceAsCode/content/files/10999024/diff_rhel9.txt)